### ISSUE TITLE
Finish attribute model training

### ISSUE THREAD

**Issue by ahalterman** [ahalterman/NGEC-2025#6]
Right now, we have the training data split into short, medium, and long docs. The long docs with batch size=2 OOMed, but the current local code now reduces that to 1 for the long docs.

The other things to do are to ablate the instructions during training: full instructions --> medium instructions --> short instructions. 
(!! Keep the output field list, though, so we can add more later). 

We also might want to add in more missing data, but that would involve re-running the synthetic generation scripts.

Finally, when training, need to halve the learning rate and double the gradient accumulation steps to prevent the noisy loss curve.

- [x] regenerate training data with complex examples
- [x] code to vary the instruction set
- [x] re-run with updated batch size, gradient accumulation, and learning rate

<img width="445" height="318" alt="Image" src="https://github.com/user-attachments/assets/3253a4da-dfd8-4b91-a986-a581cee58ebe" />
